src/extension/scripts/main.ts

```diff
@@ -109,20 +109,30 @@ chrome.tabs.onUpdated.addListener(async (tabId, changeInfo, tab) => {
 		const activations = await chrome.storage.local.get(`${PREFIX}is-enabled-${tabId}`);
 		const isEnabled = activations[`${PREFIX}is-enabled-${tabId}`];
 
-		if (changeInfo.url) {
+		const reloads = await chrome.storage.local.get(`${PREFIX}is-reload-${tab.id}`);
```

Comment:
Je remplacerai `is-reload`  par `is-reloaded` pour s’aligner avec le nommage des autres entrées.